apps/explorer/lib/explorer/chain/events/publisher.ex

```diff
@@ -22,7 +22,10 @@ defmodule Explorer.Chain.Events.Publisher do
 
   defp send_data(event_type) do
     payload = encode_payload({:chain_event, event_type})
-    @sender.send_notify(payload)
+
+    if byte_size(payload) < 7500 do
```

Comment:
can we make this an ENV var?

Reply:
or config

Reply:
Nope, it's a hardcoded value in postgres. Perhaps it could be a constant here.